packages/textrazor/textrazor.0.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "Richard Degenne <richdeg2@gmail.com>"
+authors: ["Richard Degenne <richdeg2@gmail.com>"]
+homepage: "https://github.com/Richard-Degenne/ocaml-textrazor"
+bug-reports: "https://github.com/Richard-Degenne/ocaml-textrazor/issues"
+dev-repo: "git+https://github.com/Richard-Degenne/ocaml-textrazor.git"
+license: "GPL"
+depends: [
+  "ocaml"
+  "uri"
+  "lwt_ssl"
+  "lwt_ppx"
+  "cohttp-lwt-unix"
+  "yojson"
+  "ppx_deriving_yojson"
+  "dune"{build}
+  "alcotest"{with-test}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "build" "-p" "textrazor"]
+  ["dune" "build" "-p" "textrazor" "@doc"] {with-doc}
+]
+run-test: ["dune" "runtest"]
```

Comment:
```suggestion
run-test: ["dune" "runtest" "-p" name "-j" jobs]
```
same here. Also, testing the whole thing is usually not recommanded (there might be several packages in the same repository). It's best to use `"-p" name` to avoid that